source/extensions/extensions_metadata.yaml

```diff
@@ -714,3 +714,8 @@ envoy.watchdog.profile_action:
   - envoy.guarddog_actions
   security_posture: data_plane_agnostic
   status: alpha
+envoy.dns_resolver.apple:
+  categories:
+  - envoy.network_dnsresolvers
+  security_posture: unknown
```

Comment:
This was part of the code, so it should not be alpha. I do not think we have the right posture for something like this. I think the best fit is hardened against untrusted upstreams.

Also need correct category and add the same for cares.

Reply:
ACK